Tests/AuthServiceTests/Validators/UserEmailPasswordValidatorTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using AuthenticationService.Validators;
+using DTO;
+using FluentValidation;
+using NUnit.Framework;
+using System.Linq;
+
+namespace AuthenticationServiceTests.Validators
+{
+    public class UserEmailPasswordValidatorTests
+    {
+        private UserEmailPassword emailPasswordEmpty = new UserEmailPassword(string.Empty, string.Empty);
+
```

Comment:
Why you don``t use  `var` here and below?

Reply:
Because these are fields, not variables